### DISCUSSION TITLE
2/25/21 Ring Error on latest version of home bridge and your plugin

### DISCUSSION THREAD

**Discussion by richardsonkevinb** [dgreif/ring#578] [Q&A] [answered]
Ring Error on latest version of home bridge and your plugin error below 

Failed to reach Ring server at https://oauth.ring.com/oauth/token.  urlToHttpOptions is not a function.  Trying again in 5 seconds...

I have uninstalled reinstalled plug and restated service no luck.

**Comment by dgreif** [comment]
Can you provide the version of node and homebridge you are running?

**Comment by dgreif** [comment]
Actually, looks like there is an open issue about this which indicates it should be on track to get fixed: https://github.com/dgreif/ring/issues/577

**Comment by dgreif** [comment, marked as answer]
If you install the previous version and then update to the latest version again, it should clear up the error.  One of my dependencies released a breaking change, but it has since been fixed and will self correct on after re-updating.
